R/pkg/R/sparkR.R

```diff
@@ -155,6 +155,9 @@ sparkR.sparkContext <- function(
 
   existingPort <- Sys.getenv("EXISTING_SPARKR_BACKEND_PORT", "")
   if (existingPort != "") {
+    if(sparkPackages != ""){
+        warning("--packages flag should be used with with spark-submit")
```

Comment:
@felixcheung the `sparkPackages` argument should work from the SparkR shell ? Not sure we should add that in the warning message